config/webpacker.yml

```diff
@@ -65,5 +89,8 @@ production:
   # Production depends on precompilation of packs prior to booting for performance.
   compile: false
 
+  # Extract and emit a css file
+  extract_css: true
```

Comment:
Probably don't need this given you've added as a default

Reply:
Good catch, thanks :)